doctor/crud.go

```diff
@@ -20,20 +20,19 @@ func CreateDoctor(w http.ResponseWriter, r *http.Request) {
 	decoder := json.NewDecoder(r.Body)
 	decoder.Decode(&doc)
 	if doc.Name == "" {
-		http.Error(w, "name field can not be empty", http.StatusBadRequest)
+		http.Error(w, "name field can not be empty", (400))
```

Comment:
Use variable instead of any constant
```suggestion
		http.Error(w, "name field can not be empty", http.StatusBadRequest)
```